src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests.cs

```diff
@@ -12,6 +12,7 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public partial class IOperationTests : SemanticModelTestBase
     {
+        [CompilerTrait(CompilerFeature.IOperation)]
```

Comment:
Up to you. Doesn't seem like like this file would contain tests other than IOperation...